libs/natlint/src/main/java/org/amshove/natlint/linter/LinterContext.java

```diff
@@ -75,30 +75,23 @@ public void updateEditorConfig(EditorConfig config)
 
 	void analyze(ISyntaxNode syntaxNode, IAnalyzeContext context)
 	{
-		try
```

Comment:
I think these changes are from an older `main` branch, can you update this file to the current `main`?
[The current version](https://github.com/MarkusAmshove/natls/blob/main/libs/natlint/src/main/java/org/amshove/natlint/linter/LinterContext.java#L76-L102) has this safeguard specifically mentioned [in a commit](https://github.com/MarkusAmshove/natls/commit/c385571d7428f4167c04574b8131072dc8453a74)

Reply:
This is something I changed. I now catch the error and put it in the filestatuses.csv.
So it's handled from the cli POV, but I guess I'm missing handling from the API POV.

Reply:
Looking at the current implementation in main, it doesn't continue to analyze when an exception occurs. We can ignore my comment and do graceful handling in the LSP :)